README.md

```diff
@@ -43,7 +44,13 @@ yarn hardhat node
 starknet-devnet --host 0.0.0.0
 ```
 
-### Build the cairo files
+### Build contracts in solidity
+
+```bash
+yarn hardhat compile
```

Comment:
This is done automatically when the tests are invoked

Reply:
Indeed, I was wondering why only cairo part was here. Removing it

Reply:
Indeed, I was wondering why only cairo part was here. Removing it